TEST_RUN_EVENTS.csv

```diff
@@ -0,0 +1,39 @@
+Test Run Events,,,
+,,,
+EVENT_SUFFIX,Description,,
+_INFO,Adding extra context for an specific task,,
+_SUCCESS,The task has finalized with no errors,,
+_ERROR,An execution problem was found,,
+_START,A new task section has begun,,
+,,,
+,,,
+,,,
+,,,
+,,,
+Stage  ,Event Type,Description,Definition
+Trigger,CREATED_INFO,Trigger Run has been created,
+Trigger,RESOLVE_ERROR,Resolving trigger details failed,
+Trigger,RESOLVE_SUCCESS,Resolving trigger details failed,

```

Comment:
Should this be:
Trigger,RESOLVE_SUCCESS,Successful resolving of trigger details,